src/platyPS/platyPS.psm1

```diff
@@ -419,6 +419,7 @@ function Update-MarkdownHelp
             $oldModel = $oldModels[0]
 
             $name = $oldModel.Name
+            [Array]$loadedModulesBefore = $(Get-Module | Select-Object Property Name)
```

Comment:
Should it be `Select-Object -Property Name` instead? I think that causes the tests to fail.